### ISSUE TITLE
Mac OS X 10.15.7

### ISSUE THREAD

**Issue by Diddi81Ma** [ProjectOpenSea/opensea-js#167]
Mac OS X 10.15.7
node 15.10.0
when trying to install I'm getting the below wall of errors that I can't quite understand.. anyone can help please?

```
lapt1:ae-web3@1.3 adm1$ npm install --save opensea-js
npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
npm WARN deprecated axios@0.17.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm WARN deprecated core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm ERR! code 1
npm ERR! git dep preparation failed
npm ERR! command /usr/local/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/Users/adm1/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm WARN deprecated fsevents@1.2.9: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm ERR! npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
npm ERR! npm WARN deprecated har-validator@5.1.3: this library is no longer supported
npm ERR! npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm ERR! npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm ERR! npm WARN deprecated chokidar@2.1.6: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm ERR! npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
npm ERR! npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm ERR! npm WARN deprecated request@2.88.0: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
npm ERR! npm WARN deprecated buffer@4.9.1: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
npm ERR! npm WARN deprecated axios@0.17.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm ERR! npm WARN deprecated core-js@2.6.9: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm ERR! npm ERR! code 1
npm ERR! npm ERR! path /Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command sh -c node-gyp rebuild
npm ERR! npm ERR! CXX(target) Release/obj.target/sha3/src/addon.o
npm ERR! npm ERR! gyp info it worked if it ends with ok
npm ERR! npm ERR! gyp info using node-gyp@7.1.2
npm ERR! npm ERR! gyp info using node@15.10.0 | darwin | x64
npm ERR! npm ERR! gyp info find Python using Python version 3.8.2 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
npm ERR! npm ERR! gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
npm ERR! npm ERR! gyp info spawn args [
npm ERR! npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! npm ERR! gyp info spawn args   '-f',
npm ERR! npm ERR! gyp info spawn args   'make',
npm ERR! npm ERR! gyp info spawn args   '-I',
npm ERR! npm ERR! gyp info spawn args   '/Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3/build/config.gypi',
npm ERR! npm ERR! gyp info spawn args   '-I',
npm ERR! npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! npm ERR! gyp info spawn args   '-I',
npm ERR! npm ERR! gyp info spawn args   '/Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/common.gypi',
npm ERR! npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/adm1/Library/Caches/node-gyp/15.10.0',
npm ERR! npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/adm1/Library/Caches/node-gyp/15.10.0/<(target_arch)/node.lib',
npm ERR! npm ERR! gyp info spawn args   '-Dmodule_root_dir=/Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3',
npm ERR! npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! npm ERR! gyp info spawn args   '--depth=.',
npm ERR! npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! npm ERR! gyp info spawn args   '--generator-output',
npm ERR! npm ERR! gyp info spawn args   'build',
npm ERR! npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! npm ERR! gyp info spawn args ]
npm ERR! npm ERR! No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.
npm ERR! npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
npm ERR! npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.
npm ERR! npm ERR! 
npm ERR! npm ERR! 
npm ERR! npm ERR! 
npm ERR! npm ERR! gyp info spawn make
npm ERR! npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! npm ERR! In file included from ../src/addon.cpp:4:
npm ERR! npm ERR! In file included from ../node_modules/nan/nan.h:2818:
npm ERR! npm ERR! ../node_modules/nan/nan_typedarray_contents.h:34:43: warning: 'GetContents' is deprecated: Use GetBackingStore. See http://crbug.com/v8/9908. [-Wdeprecated-declarations]
npm ERR! npm ERR!       data   = static_cast<char*>(buffer->GetContents().Data()) + byte_offset;
npm ERR! npm ERR!                                           ^
npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:5317:3: note: 'GetContents' has been explicitly marked deprecated here
npm ERR! npm ERR!   V8_DEPRECATE_SOON("Use GetBackingStore. See http://crbug.com/v8/9908.")
npm ERR! npm ERR!   ^
npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8config.h:402:39: note: expanded from macro 'V8_DEPRECATE_SOON'
npm ERR! npm ERR! # define V8_DEPRECATE_SOON(message) [[deprecated(message)]]
npm ERR! npm ERR!                                       ^
npm ERR! npm ERR! In file included from ../src/addon.cpp:9:
npm ERR! npm ERR! In file included from ../src/KeccakNISTInterface.h:17:
npm ERR! npm ERR! ../src/KeccakSponge.h:23:9: warning: 'ALIGN' macro redefined [-Wmacro-redefined]
npm ERR! npm ERR! #define ALIGN __attribute__ ((aligned(32)))
npm ERR! npm ERR!         ^
npm ERR! npm ERR! /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/i386/param.h:83:9: note: previous definition is here
npm ERR! npm ERR! #define ALIGN(p)        __DARWIN_ALIGN(p)
npm ERR! npm ERR!         ^
npm ERR! npm ERR! ../src/addon.cpp:83:11: error: no matching member function for call to 'Set'
npm ERR! npm ERR!                 target->Set(className, f);
npm ERR! npm ERR!                 ~~~~~~~~^~~
npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:3716:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
npm ERR! npm ERR!   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
npm ERR! npm ERR!                                     ^
npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:3719:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
npm ERR! npm ERR!   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
npm ERR! npm ERR!                                     ^
npm ERR! npm ERR! 2 warnings and 1 error generated.
npm ERR! npm ERR! make: *** [Release/obj.target/sha3/src/addon.o] Error 1
npm ERR! npm ERR! gyp ERR! build error 
npm ERR! npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
npm ERR! npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:378:20)
npm ERR! npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! npm ERR! gyp ERR! System Darwin 19.6.0
npm ERR! npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! npm ERR! gyp ERR! cwd /Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3
npm ERR! npm ERR! gyp ERR! node -v v15.10.0
npm ERR! npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! npm ERR! gyp ERR! not ok
```

__Originally posted by @jamboj in https://github.com/ProjectOpenSea/opensea-js/issues/69__

**Comment by 0x7An** [comment]
@Diddi81Ma Installation instructions say **We recommend switching to Node.js version 8.11**, I'm using the v12.22.4 here, and is working, you should downgrade your node version.

**Comment by Diddi81Ma** [comment]
Ich versuche es

**Comment by Diddi81Ma** [comment]
Danke

**Comment by Diddi81Ma** [comment]
> Mac OS X 10.15.7
> 
> node 15.10.0
> 
> when trying to install I'm getting the below wall of errors that I can't quite understand.. anyone can help please?
> 
> 
> 
> ```
> 
> lapt1:ae-web3@1.3 adm1$ npm install --save opensea-js
> 
> npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
> 
> npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
> 
> npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
> 
> npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
> 
> npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
> 
> npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
> 
> npm WARN deprecated axios@0.17.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
> 
> npm WARN deprecated core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
> 
> npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
> 
> npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
> 
> npm ERR! code 1
> 
> npm ERR! git dep preparation failed
> 
> npm ERR! command /usr/local/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/Users/adm1/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit
> 
> npm ERR! npm WARN using --force Recommended protections disabled.
> 
> npm ERR! npm WARN deprecated fsevents@1.2.9: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
> 
> npm ERR! npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
> 
> npm ERR! npm WARN deprecated har-validator@5.1.3: this library is no longer supported
> 
> npm ERR! npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
> 
> npm ERR! npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
> 
> npm ERR! npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
> 
> npm ERR! npm WARN deprecated chokidar@2.1.6: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
> 
> npm ERR! npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
> 
> npm ERR! npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
> 
> npm ERR! npm WARN deprecated request@2.88.0: request has been deprecated, see https://github.com/request/request/issues/3142
> 
> npm ERR! npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
> 
> npm ERR! npm WARN deprecated buffer@4.9.1: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
> 
> npm ERR! npm WARN deprecated axios@0.17.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
> 
> npm ERR! npm WARN deprecated core-js@2.6.9: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
> 
> npm ERR! npm ERR! code 1
> 
> npm ERR! npm ERR! path /Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3
> 
> npm ERR! npm ERR! command failed
> 
> npm ERR! npm ERR! command sh -c node-gyp rebuild
> 
> npm ERR! npm ERR! CXX(target) Release/obj.target/sha3/src/addon.o
> 
> npm ERR! npm ERR! gyp info it worked if it ends with ok
> 
> npm ERR! npm ERR! gyp info using node-gyp@7.1.2
> 
> npm ERR! npm ERR! gyp info using node@15.10.0 | darwin | x64
> 
> npm ERR! npm ERR! gyp info find Python using Python version 3.8.2 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
> 
> npm ERR! npm ERR! gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
> 
> npm ERR! npm ERR! gyp info spawn args [
> 
> npm ERR! npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
> 
> npm ERR! npm ERR! gyp info spawn args   'binding.gyp',
> 
> npm ERR! npm ERR! gyp info spawn args   '-f',
> 
> npm ERR! npm ERR! gyp info spawn args   'make',
> 
> npm ERR! npm ERR! gyp info spawn args   '-I',
> 
> npm ERR! npm ERR! gyp info spawn args   '/Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3/build/config.gypi',
> 
> npm ERR! npm ERR! gyp info spawn args   '-I',
> 
> npm ERR! npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
> 
> npm ERR! npm ERR! gyp info spawn args   '-I',
> 
> npm ERR! npm ERR! gyp info spawn args   '/Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/common.gypi',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dvisibility=default',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/adm1/Library/Caches/node-gyp/15.10.0',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/adm1/Library/Caches/node-gyp/15.10.0/<(target_arch)/node.lib',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dmodule_root_dir=/Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Dnode_engine=v8',
> 
> npm ERR! npm ERR! gyp info spawn args   '--depth=.',
> 
> npm ERR! npm ERR! gyp info spawn args   '--no-parallel',
> 
> npm ERR! npm ERR! gyp info spawn args   '--generator-output',
> 
> npm ERR! npm ERR! gyp info spawn args   'build',
> 
> npm ERR! npm ERR! gyp info spawn args   '-Goutput_dir=.'
> 
> npm ERR! npm ERR! gyp info spawn args ]
> 
> npm ERR! npm ERR! No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.
> 
> npm ERR! npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
> 
> npm ERR! npm ERR! No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.
> 
> npm ERR! npm ERR! 
> 
> npm ERR! npm ERR! 
> 
> npm ERR! npm ERR! 
> 
> npm ERR! npm ERR! gyp info spawn make
> 
> npm ERR! npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
> 
> npm ERR! npm ERR! In file included from ../src/addon.cpp:4:
> 
> npm ERR! npm ERR! In file included from ../node_modules/nan/nan.h:2818:
> 
> npm ERR! npm ERR! ../node_modules/nan/nan_typedarray_contents.h:34:43: warning: 'GetContents' is deprecated: Use GetBackingStore. See http://crbug.com/v8/9908. [-Wdeprecated-declarations]
> 
> npm ERR! npm ERR!       data   = static_cast<char*>(buffer->GetContents().Data()) + byte_offset;
> 
> npm ERR! npm ERR!                                           ^
> 
> npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:5317:3: note: 'GetContents' has been explicitly marked deprecated here
> 
> npm ERR! npm ERR!   V8_DEPRECATE_SOON("Use GetBackingStore. See http://crbug.com/v8/9908.")
> 
> npm ERR! npm ERR!   ^
> 
> npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8config.h:402:39: note: expanded from macro 'V8_DEPRECATE_SOON'
> 
> npm ERR! npm ERR! # define V8_DEPRECATE_SOON(message) [[deprecated(message)]]
> 
> npm ERR! npm ERR!                                       ^
> 
> npm ERR! npm ERR! In file included from ../src/addon.cpp:9:
> 
> npm ERR! npm ERR! In file included from ../src/KeccakNISTInterface.h:17:
> 
> npm ERR! npm ERR! ../src/KeccakSponge.h:23:9: warning: 'ALIGN' macro redefined [-Wmacro-redefined]
> 
> npm ERR! npm ERR! #define ALIGN __attribute__ ((aligned(32)))
> 
> npm ERR! npm ERR!         ^
> 
> npm ERR! npm ERR! /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/i386/param.h:83:9: note: previous definition is here
> 
> npm ERR! npm ERR! #define ALIGN(p)        __DARWIN_ALIGN(p)
> 
> npm ERR! npm ERR!         ^
> 
> npm ERR! npm ERR! ../src/addon.cpp:83:11: error: no matching member function for call to 'Set'
> 
> npm ERR! npm ERR!                 target->Set(className, f);
> 
> npm ERR! npm ERR!                 ~~~~~~~~^~~
> 
> npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:3716:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
> 
> npm ERR! npm ERR!   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
> 
> npm ERR! npm ERR!                                     ^
> 
> npm ERR! npm ERR! /Users/adm1/Library/Caches/node-gyp/15.10.0/include/node/v8.h:3719:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
> 
> npm ERR! npm ERR!   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
> 
> npm ERR! npm ERR!                                     ^
> 
> npm ERR! npm ERR! 2 warnings and 1 error generated.
> 
> npm ERR! npm ERR! make: *** [Release/obj.target/sha3/src/addon.o] Error 1
> 
> npm ERR! npm ERR! gyp ERR! build error 
> 
> npm ERR! npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
> 
> npm ERR! npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
> 
> npm ERR! npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:378:20)
> 
> npm ERR! npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
> 
> npm ERR! npm ERR! gyp ERR! System Darwin 19.6.0
> 
> npm ERR! npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
> 
> npm ERR! npm ERR! gyp ERR! cwd /Users/adm1/.npm/_cacache/tmp/git-clone-74cc97c0/node_modules/sha3
> 
> npm ERR! npm ERR! gyp ERR! node -v v15.10.0
> 
> npm ERR! npm ERR! gyp ERR! node-gyp -v v7.1.2
> 
> npm ERR! npm ERR! gyp ERR! not ok
> 
> ```
> 
> __Originally posted by @jamboj in https://github.com/ProjectOpenSea/opensea-js/issues/69__

**Comment by esteban-OpenSea** [comment]
Please use the latest version of Node and the SDK. If that doesn't work try installing with yarn.

```
npm install --global yarn
yarn add opensea-js
```

Please follow this issue for the latest updates on the npm install: https://github.com/ProjectOpenSea/opensea-js/issues/471
